sk0-front/src/app/app.routes.ts

```diff
@@ -1,13 +1,17 @@
-import { Routes } from '@angular/router';
-
-export const routes: Routes = [
-  {
-    path: '',
-    redirectTo: 'home',
-    pathMatch: 'full',
-  },
-  {
-    path: 'home',
-    loadComponent: () => import('./home/home.page').then((m) => m.HomePage),
+import { Routes } from '@angular/router';
+
+export const routes: Routes = [
+  {
+    path: '',
+    redirectTo: 'home',
+    pathMatch: 'full',
+  },
+  {
+    path: 'home',
+    loadComponent: () => import('./home/home.page').then((m) => m.HomePage),
+  },
  {
```

Comment:
```suggestion
  }, {
```

Reply:
Ez valamilyen prettier beállítás? Bevallom, nem igazán tudom hogy tudnám megváltoztatni...